easysocial/easysocial/friends.php

```diff
@@ -69,6 +69,24 @@ function getFriends()
 							$flag=0;						
 							$mssg=JText::_( 'PLG_API_EASYSOCIAL_NOT_SENT_REQUEST' );	
 			break;
+			case 'blocked': //getting block friends	added by pratiksha 
+							$res =  $model->getBlockedUsers($userid);
+
```

Comment:
$res will already give you all the details of the blocked users. Why are you discarding the information, keeping the ids only and then again getting all the information based on ids ??